podman/Dockerfile-podman

```diff
@@ -0,0 +1,11 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+RUN subscription-manager register --username='...' --password='...' --name=ubi8-ansible-4
```

Comment:
@guidograzioli can we add variable instead of ..., this way ```podman build . ``` will fail if those values are not provided.

Also, we need change ubi8-ansible-4 to something else (ubi8-ansible)

Reply:
will do